src/components/ui/bottomsheet.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useRef, useEffect } from 'react';
+import {
+  View,
+  Modal,
+  Animated,
+  PanResponder,
+  TouchableWithoutFeedback,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+import { styled } from 'nativewind';
+import { cn } from '../../lib/utils';
+
+interface BottomSheetProps {
+  visible: boolean;
+  onClose: () => void;
+  children?: React.ReactNode;
+  containerClassName?: string;
+  backgroundColor?: string;
+  borderRadius?: string;
+  height?: number;
```

Comment:
You do not need it, if you are using cn()

Reply:
cn don't work for that